packages/manager/apps/key-management-service/src/components/credential/credentialDatagrid/credentialDatagridCells.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import {
+  DataGridClipboardCell,
+  DataGridTextCell,
+  Links,
+} from '@ovhcloud/manager-components';
+import { OkmsCredential } from '@/types/okmsCredential.type';
+import { useFormattedDate } from '@/hooks/useFormattedDate';
+import { CredentialStatus } from '../credentialStatus/credentialStatus.component';
+
+export const DatagridCredentialCellName = (credential: OkmsCredential) => {
+  const navigate = useNavigate();
+  return (
+    <div>
```

Comment:
Can't we return < Links > directly ? Or is there a side effect where we have to wrap it ?